submissions/5mountains/popup/index.html

```diff
@@ -0,0 +1,150 @@
+                        <a class="submenu__link" href="#">
+                            <img src="./img/submenu/news.png" alt="news icon">
+                            News
+                        </a>
+                    </li>
+                    <li class="submenu__item">
+                        <a class="submenu__link" href="#">
+                            <img src="./img/submenu/maps.png" alt="map icon">
+                            Map
+                        </a>
+                    </li>
+                    <li class="submenu__item">
+                        <a class="submenu__link" href="#">
+                            <img src="./img/submenu/files.png" alt="files icon">
+                            Files
+                        </a>
+                    </li>
+                    <li class="submenu__item">
+                        <a class="submenu__link" href="#">
+                            <img src="./img/submenu/contacts.png" alt="contacts icon">
+                            Contacts
+                        </a>
+                    </li>
+                    <li class="submenu__item">
+                        <a class="submenu__link" href="#">
+                            <img src="./img/submenu/translation.png" alt="translation icon">
+                            Translate
+                        </a>
+                    </li>
+                    <li class="submenu__item">
+                        <a class="submenu__link" href="#">
+                            <img src="./img/submenu/videos.png" alt="videos icon">
+                            Videos
+                        </a>
+                    </li>
+                    <li class="submenu__item">
+                        <a class="submenu__link" href="#">
+                            <img src="./img/submenu/search.png" alt="search icon">
+                            Search
+                        </a>
+                    </li>
+                    <li class="submenu__item">
+                        <a class="submenu__link" href="#">
+                            <img src="./img/submenu/photos.png" alt="photos icon">
+                            Photos
+                        </a>
+                    </li>
+                    <li class="submenu__item">
+                        <a class="submenu__link" href="#">
+                            <img src="./img/submenu/apps.png" alt="apps icon">
+                            Apps
+                        </a>
+                    </li>
+                    <li class="submenu__item">
+                        <a class="submenu__link" href="#">
+                            <img src="./img/submenu/documents.png" alt="documents icon">
+                            Documents
+                        </a>
+                    </li>
+                    <input type="checkbox" id="submenu__input" class="submenu__item-input">
+                    <label for="submenu__input" class="submenu__item-label">More</label>
```

Comment:
Your `ul` element should contain only `li` tags. But I see that you put `input` and `label` to it.

Reply:
Yes, I know that extra ul should contain only on li tags (like submenu), but I thought that to solve such a task, the rules can be broken...

Reply:
You can just wrap them into `li` tag and it will be OK